Makefile

```diff
@@ -12,13 +12,13 @@ lint:
 # Rust
 	flatpak run --command="/usr/lib/sdk/rust-stable/bin/rustfmt" --filesystem=host re.sonny.Workbench --check --edition 2021 demos/**/*.rs
 # Python
-	flatpak run --command="black" --filesystem=host re.sonny.Workbench --check demos/**/*.py
+	flatpak run --command="ruff" --filesystem=host re.sonny.Workbench check --config=../src/langs/python/ruff.toml demos/**/*.py
```

Comment:
I realize this may be an issue when running via GitHub Actions.
Instead I could copy the ruff.toml in here and we use 2 different files as you initially suggested. Does that sound good?